mlflow/tracking/fluent.py

```diff
@@ -1329,19 +1333,30 @@ def print_auto_logged_info(r):
         "pytorch_lightning": pytorch.autolog,
     }
 
-    def get_autologging_params(autolog_fn):
+    def get_autologging_params(integration, autolog_fn):
         try:
             needed_params = list(inspect.signature(autolog_fn).parameters.keys())
-            return {k: v for k, v in locals_copy if k in needed_params}
+            param_spec = inspect.signature(autolog_fn).parameters
+            default_params = {param.name: param.default for param in param_spec.values()}
+            return {k: AutologgingConfigManager.get_config(integration, k, default_params[k]) for k in needed_params}
         except Exception:
             return {}
 
+    def set_mlflow_autolog_params_in_config(integration):
+        try:
+            needed_params = list(inspect.signature(autolog).parameters.keys())
+            [AutologgingConfigManager.set_mlflow_config(integration, k, v)
+             for k, v in locals_copy if k in needed_params]
+        except Exception:
+            pass
+
```

Comment:
Following up on above comment, this new function gathers the arguments of `mlflow.autolog` and sets them in the config manager, key-ed by the integration name.